Master-OB-OpenAPI.json

```diff
@@ -18894,6 +18914,35 @@
           "description": ""
         }
       }
+    },
+    "ChecksumTypeItemType": {
+      "description": "",
+      "enums": {
+        "CRC": {
+          "label": "CRC",
+          "description": "Cyclical Redundancy Check - NIST SP 800-72"
```

Comment:
This defines the term Cyclical Redundancy Check but there are literally dozens of such checks. Knowing it's a CRC is probably not enough information to decode a checksum.